spec/models/profile_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Profile, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { should have_valid(:user_id).when('1', '100') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.